src/pages/questionPage.js

```diff
@@ -18,10 +21,58 @@ export const initQuestionPage = () => {
   userInterface.appendChild(questionElement);
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
+  answersListElement.classList.add('answers-list');
+
+  // This block displays current score
+  const currentScores = document.createElement('div');
+  currentScores.classList.add('current-scores');
+  currentScores.innerHTML = `Correct answers: ${correctCounter} of ${quizData.questions.length}`;
+  userInterface.appendChild(currentScores);
```

Comment:
Again, this was not part of the **must haves** we discussed.

Reply:
This part of the code looks like something that belongs in the `/views` folder.

You are defining user elements and in MVC that it that responsibility of the view layer.

Reply:
Ok, deleted this code